app/cronjob/trailingTrade/step/ensure-grid-trade-order-executed.js

```diff
@@ -248,6 +248,8 @@ const execute = async (logger, rawData) => {
         },
         temporaryDisableActionAfterConfirmingOrder
       );
+
+      PubSub.publish('check-open-orders', {});
```

Comment:
I think we should add to SELL filled order to check open orders as well.
I will add in.

Reply:
Hello @chrisleekr 
I am not sure if we really need it in SELL filled order as well.
We add this only in BUY filled order to check the rest open order and cancel them if we reached the limit.

Reply:
You are absolutely right.
Scratch my comment.